hil/cli.py

```diff
@@ -853,5 +854,7 @@ def main():
             sys.exit('Error: %s\n' % e.message)
         except BadArgumentError as e:
             sys.exit('Error: %s\n' % e.message)
+        except SchemaError as e:
+            sys.exit('Error in config file: %s\n' % e.message)
```

Comment:
As we discussed during the weekly meeting. Probably we should just return the `e`? Since `e.message` might be `None?